src/common.c

```diff
@@ -443,44 +443,35 @@ get_mentions(gboolean whole_word, gboolean case_sensitive, const char* const mes
 }
 
 gboolean
-call_external(gchar** argv, gchar** std_out, gchar** std_err)
+call_external(gchar** argv, gint* std_out, gint* std_in, gint* std_error, GPid* pid)
```

Comment:
What's the point of adding those two parameters and then not using them?

IMO the better change here would be to remove those two parameters that are unused.

Reply:
Should I be removing the 4 parameters in the new `call_external`?

Reply:
Yes, it looks like they are not used anywhere.

Reply:
Sure :3